swagger.yaml

```diff
@@ -10,27 +10,461 @@ basePath: "/v1"
+      produces:
+        - application/json
+      responses:
+        501:
+          description: Not Implemented
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/VersionsList'
+
+  /bulletins/{id}/editions/{edition}/versions/{version}:
+    get:
+      tags:
+        - "Public"
+      summary: Get a version
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/edition'
+        - $ref: '#/parameters/version'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/Version'
+
+parameters:
+  id:
+    name: id
+    description: ID of a bulletin
+    type: string
+    in: path
+    required: true
+  edition:
+    name: id
+    description: Edition of a bulletin
+    type: string
+    in: path
+    required: true
+  version:
+    name: id
+    description: Version of a bulletin
+    type: string
+    in: path
+    required: true
+  limit:
+    name: limit
+    description: The number of items requested, defaulted to 50 and limited to 1000.
+    type: integer
+    required: false
+    in: query
+  offset:
+    name: offset
+    description: The first row of resources to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter.
+    type: integer
+    required: false
+    in: query
 
 definitions:
-  HelloResponse:
+  BulletinsList:
+    type: object
+    properties:
+      count:
+        $ref: '#/definitions/itemCount'
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/Bulletin'
+
+  Bulletin:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/id'
+      contacts:
+        type: array
+        items:
+          $ref: '#/definitions/Contact'
+      description:
+        $ref: '#/definitions/summary'
+      keywords:
+        type: array
+        description: List of keywords relating to this bulletin
+        items:
+          type: string
+        example: [ "work" , "jobless" ]
+      license:
+        description: "The standard Government license right text for the bulletin"
+        type: string
+      national_statistic:
+        $ref: '#/definitions/nationalStatistic'
+      next_release:
+        $ref: '#/definitions/nextRelease'
+      related_links:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_data:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_documents:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodologies:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_methodology_articles:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
+      related_topics:
+        type: array
+        items:
+          $ref: '#/definitions/RelatedLink'
```

Comment:
I've moved these from the version level to the bulletin level as discussed. I've also renamed the `links` and `topics` fields for consistency but there was an open comment on the spreadsheet as to what to do with them. Please advise.